src/mock_crust/crust.rs

```diff
@@ -73,13 +73,13 @@ impl Service {
     /// Start service discovery (beacon).
     /// Note: beacon is not yet implemented in mock.
     pub fn start_service_discovery(&mut self) {
-        trace!("[MOCK] start_service_discovery not implemented in mock");
+        trace!(target: "mock_crust", "[MOCK] start_service_discovery not implemented in mock");
```

Comment:
I changed half-way through, to avoid confusion with the real crust. But I guess we'll never use both at the same time.